AWSAuthSDK/Sources/AWSMobileClient/AWSMobileClientExtensions.swift

```diff
@@ -118,6 +107,23 @@ extension AWSMobileClient: AWSIdentityProviderManager {
         let task = AWSTask.init(result: dict as NSDictionary)
         return task
     }
+    
+    func setLoginMapTaskUsing(tokens: Tokens?, error: Error?, to task: AWSTaskCompletionSource<NSDictionary>) {
```

Comment:
nit: naming is a bit awkward.
```suggestion
    func setLoginMapTokensResult(on task: AWSTaskCompletionSource<NSDictionary>, tokens: Tokens?, error: Error?) {
```

Reply:
renamed